runner/config/default.js

```diff
@@ -104,7 +104,7 @@ module.exports = {
   // Control which is used. Accepts "test" | "production" | "".
   apiEnv: "",
   payApiUrl: "https://publicapi.payments.service.gov.uk/v1",
-  documentUploadApiUrl: "http://localhost:9000",
+  documentUploadApiUrl: null,
```

Comment:
sorry, mind if we comment out instead to follow the pattern of the rest of the config? 
```suggestion
  // documentUploadApiUrl: "http://localhost:9000",
```